.github/actions/windows-run_examples/action.yml

```diff
@@ -19,8 +19,16 @@ runs:
         $errchk = (Get-ChildItem -Path ./| Where-Object { $_.Name -like "*_example.exe"} | Measure-Object).Count
         if( $errchk -gt 0){
           Write-Output "Running windows Examples."
-          Get-ChildItem -Path ./| Where-Object { $_.Name -like "*_example.exe"} | Foreach { Write "Running $_"; if( $(Start-Process $_.FullName -Passthru -Wait).ExitCode -ne 0){Write "Programm terminated with error, abort."; Exit 1}}
```

Comment:
Was the `-Passthru` intended to pipe out the output? Do you know why it did not work, or why it is not needed with the `&`?

Reply:
From my understanding, its used to return the process object to check the exit code but does not forward output. There would also be flags to make this possible,  however, i prefer the current solution.